code/CreateAddedItems.js

```diff
@@ -0,0 +1,16 @@
+//CreateItem
+exports.function = function(addedItem, addedItem2, addedItem3) {
+    items = []
+    if (addedItem) {
+      items.push(addedItem)
```

Comment:
don't need to delete type for first one?

Reply:
All elements of the output are of type Item, so we're good with this one